flare/framework/tests/test_recurrent.py

```diff
@@ -146,7 +152,7 @@ def test_hierchical_sequences(self):
         ## 2. We take the last output of the words and the word states
         ## 3. In the higher level, we multiply the last word output with the sentence state,
```

Comment:
we multiply the last word output with the sentence state: this comment should be updated with something like "added with the mean of static img input"